.github/workflows/cross-compile.yml

```diff
@@ -5,8 +5,6 @@ on:
     branches: [ criu-dev ]
   pull_request:
     branches: [ criu-dev ]
-  schedule:
-    - cron:  '55 5 * * *'
```

Comment:
why do we need to move this in a new file?

Reply:
Maybe because I was not able to correctly understand the documentation :wink: 

What I wanted to do is set up a daily build for `master` and `criu-dev`. The problem is that it is not possible, as far as I understood it, to specify a branch in the `schedule` section. To have a daily build on two branches I have to use the matrix thing like I did and change the checkout section to use that. This means, that I am no longer checking out the PR this is supposed to run, but always `master` and `criu-dev`. Something like this.

From my understanding it is basically a short-coming that I can tell which actions to run the schedule on easily. I am also not happy to have the same content, what to build, in two files now, but I have not found a better solution.